rubicon_ml/repository/base.py

```diff
@@ -13,6 +14,17 @@
 from rubicon_ml.exceptions import RubiconException
 from rubicon_ml.repository.utils import json, slugify
 
+Domains = TypeVar(
```

Comment:
can we put this in the domain init or something? if we need it elsewhere, importing from here might cause circular dependency issues

Reply:
Good call, yes.